usersimcrs/items/item_collection.py

```diff
@@ -1,8 +1,8 @@
 """Represents a collection of items.
 
-Items are characterized by a set of properties, which correspond to slots in a
-domain. Items and can be loaded from CSV by providing a mapping from CSV fields
-to properties (i.e., domain slots).
+Items are characterized by a set of properties, which correspond to
+slots in a domain. Items and can be loaded from CSV by providing a
+mapping from CSV fields to properties (i.e., domain slots).
```

Comment:
Did something happen with the black configuration? The original text was within the 80 chars limit.

Reply:
I did not change the configuration, so not sure what happened here. I reverted these changes.